projects/angular/src/accordion/_accordion.clarity.scss

```
@@ -268,18 +269,21 @@
 
     .clr-accordion-header-button {
       align-items: center;
-      padding: #{tokens.$cds-global-space-5} #{tokens.$cds-global-space-6};
+      padding: #{density.$clr-base-vertical-offset-m} #{density.$clr-base-horizontal-offset-l};
     }
 
     .clr-accordion-content {
       padding: 0;
     }
 
     .clr-accordion-inner-content {
-      padding: tokens.$cds-global-space-5 tokens.$cds-global-space-6 tokens.$cds-global-space-5
-        tokens.$cds-global-space-11;
+      padding: density.$clr-base-vertical-offset-m density.$clr-base-horizontal-offset-l
+        density.$clr-base-vertical-offset-m
+        calc(
+          density.$clr-base-horizontal-offset-l + density.$clr-base-icon-size-s + density.$clr-base-horizontal-offset-m
+        );
       color: accordion-variables.$clr-accordion-content-color;
-      font-size: accordion-variables.$clr-accordion-content-font-size;
+      @include mixins.generate-typography-token('SECONDARY-13-RG-STD');
```

The accordion `clr-accordion-content-font-size` property may be deprecated, but until v18 we need it.
Use it as override and replace it's value with the correct density `font-size` token.

```suggestion
      @include mixins.generate-typography-token('SECONDARY-13-RG-STD', (
        font-size: accordion-variables.$clr-accordion-content-font-size
      ));
```

---

Fixed in [f717d83](https://github.com/vmware-clarity/ng-clarity/pull/1874/commits/f717d8345014a22ecdd7b5c5418539ad15967172)